src/Postnl.php

```diff
@@ -329,6 +329,61 @@ public function currentStatus($barcode)
     }
 
     /**
+     * @param $postalCode
+     * @param string $allowSundaySorting
+     * @param null|string $deliveryDate
+     * @param string $countryCode
+     * @return ComplexTypes\GetNearestLocationsResponse
+     */
+    public function getNearestLocation(
+        $postalCode,
+        $allowSundaySorting = 'false',
+        $deliveryDate = null,
+        $countryCode = 'NL'
+    ) {
+        $message = new ComplexTypes\Message;
+        $location = new ComplexTypes\Location($postalCode, $allowSundaySorting, $deliveryDate);
+
+        $request = new ComplexTypes\GetNearestLocationsRequest($message, $location, $countryCode);
+        return $this->call('LocationClient', __FUNCTION__, $request);
+    }
+
+    /**
+     * @param string $postalCode
+     * @param string $houseNumber
+     * @param array $options
+     * @param string $startDate
+     * @param string $endDate
+     * @param string $countryCode
+     * @param string $allowSundaySorting
+     * @return ComplexTypes\GetTimeframesResponse
+     * @throws ComplexTypes\CifException
+     * @throws SoapFault
+     */
+    public function getTimeframes(
```

Comment:
The other clients also have a corresponding `Postnl::setFooClient` method, which allows separately instantiating client `Foo`, instead of letting the main `Postnl` class handle it. I don't know if this is of practical value yet, but for now I'd like to keep stuff consistent.